src/Exception/RoutesPathNotFoundException.php

```diff
@@ -1,18 +1,33 @@
 <?php
 
+declare(strict_types=1);
+
 namespace BlankFramework\FilePathRouter\Exception;
 
 use Exception;
 
-class RoutesPathNotFoundException extends \Exception
+/**
+ * Exception thrown when a routes path cannot be found.
+ */
+final class RoutesPathNotFoundException extends Exception
 {
-    public function __construct(?string $path = null)
+    /**
+     * Constructs a new RoutesPathNotFoundException.
+     *
+     * This exception is thrown when the path specified for the routes cannot be found in the system.
+     * If a specific path is provided, the exception message includes this path.
+     * If no path is given, a generic error message is used.
+     *
+     * @param null|string $path The path to the routes that could not be found. Optional.
+     */
+    public function __construct(string $path = null)
```

Comment:
To confirm if this will work I created a small CLI command. The above code is correct.

`<?php

function checkIfIsNull(string $path = null)
{
    if (is_null($path)) {
        return 'path is null';
    }

    return 'path is not null';
}


print checkIfIsNull().PHP_EOL;
print checkIfIsNull('Test').PHP_EOL;`